### DISCUSSION TITLE
Logger Question

### DISCUSSION THREAD

**Discussion by troysjanda** [uBlockOrigin/uBlock-issues#3213] [General] [answered]
I just noticed that in the logger when clicking on a line it is docked at the bottom and not a floating window as before. Is there a way to toggle this back to the window mode instead of the bottom docked? see image. 

![question](https://github.com/uBlockOrigin/uBlock-issues/assets/24624945/e426e81d-6d00-4d3d-a6a6-a48beea60223)

**Comment by gorhill** [comment, marked as answer]
No, I removed the floating window mode.

**Reply by troysjanda** [reply]
Alright ty for the reply much appreciated.
